src/main/java/uk/gov/hmcts/juror/api/moj/controller/JurorRecordController.java

```diff
@@ -434,6 +436,39 @@ public ResponseEntity<JurorAttendanceDetailsResponseDto> getJurorAttendanceDetai
         return ResponseEntity.ok().body(details);
     }
 
+
+    /**
+     * Get the attendance and payments log details for a given Juror.
+     *
+     * @param jurorNumber Unique Juror number of the juror
+     * @return Fully populated DTO of the juror's attendance and payments log
+     */
+    @GetMapping(path = "/{jurorNumber}/payments")
+    @Operation(summary = "Get attendance and payments log for a given juror")
```

Comment:
Do we need to restrict to court/bureau user types?

Reply:
This one is now restricted to court only